src/systems/abstractsystem.jl

```diff
@@ -870,6 +870,11 @@ function namespace_guesses(sys)
     Dict(unknowns(sys, k) => namespace_expr(v, sys) for (k, v) in guess)
 end
 
+function namespace_parameter_dependencies(sys)
+    pdeps = get_parameter_dependencies(sys)
+    Dict(dependent_parameters(sys, k) => namespace_expr(v, sys) for (k, v) in pdeps)
```

Comment:
```suggestion
    Dict(parameters(sys, k) => namespace_expr(v, sys) for (k, v) in pdeps)
```

No need to create a new method that does the same thing as an old one